### DISCUSSION TITLE
Docs are now live with light/darkmode

### DISCUSSION THREAD

**Discussion by Levdbas** [timber/timber#3037] [Announcements]
Thanks to the hard work of @heybran we now have the docs with a dark/light mode switch as well. Give it a spin and let us know what you think!

https://timber.github.io/docs/v2/

**Comment by martinpl** [comment]
Nice, light mode works better for eink users, thanks :)

**Comment by gchtr** [comment]
Fantastic! Huge thanks to @heybran for all the work 🙌.

**Reply by nlemoine** [reply]
Great job! Thanks @heybran ! 🌓🌓🌓

**Comment by heybran** [comment]
You folks literally encouraged me to invest more time into open source. Thank you all!
